charts/develop/jdownloader2/README.md

```diff
@@ -0,0 +1,44 @@
+# Introduction
```

Comment:
you can safely ignore all readme's and .md files, they are autogenerated.

Reply:
do I remove them?

Reply:
done